### PR TITLE
Revert deadline reduction

### PR DESCRIPTION
**Pull request by anxolin**
# Summary
Bring back the deadline options. 

Reverts  3fc3ecf1f0cc1d38a6aad3457428fccc162fbcba from #1625

Please, let's only merge once @nlordell has approved is fine to do so. 

Also, @MindyCoW what do you think should be the default? Currently 7 days

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]

**Comment by elena-zh** [comment]
Can't test: no preview link :(

File: src/cow-react/modules/limitOrders/pure/DeadlineSelector/deadlines.ts [hunk 1]
```diff
@@ -7,11 +7,14 @@ export interface LimitOrderDeadline {
 
 export const maxCustomDeadline = ms`1y`
```
Comment (nlordell) [inline src/cow-react/modules/limitOrders/pure/DeadlineSelector/deadlines.ts:8]: I think we settled on 6mo for this.

**Review by nlordell — APPROVED** [review]
LGTM - I think the `maxCustomDeadline` is high though.

File: src/cow-react/modules/limitOrders/pure/DeadlineSelector/deadlines.ts (hunk 1 above)
Comment (shoom3301) [reply src/cow-react/modules/limitOrders/pure/DeadlineSelector/deadlines.ts]: Thanks! Changed to 6 months

**Pull request merged**